infra/aws/e2e/accounts/tf/accounts.tf

```diff
@@ -0,0 +1,182 @@
+terraform {
+  required_version = "~> 0.13"
+  backend "gcs" {
+    bucket = "k8s-infra-clusters-terraform"
+    prefix = "aws/accounts"
+    # NOTE: Must specify encryption key
+  }
+}
+
+
+# Terraform doesn't work well with count in modules (even in 0.13)
+
+# Accounts for cluster-api-provider-aws project
+module "capa-user-00" {
+  source      = "./modules/account-and-user"
```

Comment:
Style Nit: For these module instantiations, you might be able to [take advantage of the `for_each` command](https://www.terraform.io/docs/configuration/meta-arguments/for_each.html) and move these to be variables.